### PR TITLE
cleanup: Error string fmt and handle error

### PR DESCRIPTION
**Pull request by falser101**
Error string fmt and handle error
Signed-off-by: jianfei.zhang <jianfei.zhang@daocloud.io>

**What type of PR is this?**
/kind cleanup
**What this PR does / why we need it**:
 Error string fmt and handle error
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: cloud/pkg/admissioncontroller/admit_rule.go [hunk 1]
```diff
@@ -41,8 +41,8 @@ func admitRule(review admissionv1beta1.AdmissionReview) *admissionv1beta1.Admiss
 		reviewResponse.Allowed = true
 		return &reviewResponse
 	default:
-		err := fmt.Errorf("Unsupported webhook operation %v", review.Request.Operation)
-		klog.Errorf("Unsupported webhook operation %v", review.Request.Operation)
+		err := fmt.Errorf("unsupported webhook operation %v", review.Request.Operation)
+		klog.Errorf("unsupported webhook operation %v", review.Request.Operation)
```
Comment (Shelley-BaoYue) [inline cloud/pkg/admissioncontroller/admit_rule.go:45]: I think it's better to keep the first letter capitalized in klog.

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

File: cloud/pkg/admissioncontroller/admit_rule.go (hunk 1 above)
Comment (zc2638) [reply cloud/pkg/admissioncontroller/admit_rule.go]: I agree with you.
But it looks like @fisherxu doesn't think so.

Comment (zc2638) [reply cloud/pkg/admissioncontroller/admit_rule.go]: ref: https://github.com/kubeedge/kubeedge/pull/4011/files#diff-364f053b5a428af19fb62f64c0c1675599b8d5ddc316c75bc71e25fc6340e54aR36

**Pull request merged**